### ISSUE TITLE
Length of future trajectory for Motion Prediction (80)

### ISSUE THREAD

**Issue by arbaazkhan2** [waymo-research/waymo-open-dataset#296]
Is there a reason the authors decided to set the length of the time horizon for prediction to 80 as shown in the tutorial? Intuitively would it not make sense to predict for a smaller horizon say 10 or 20? 

Does changing the time horizon for the prediction trajectory somehow affect the testing code?

**Comment by scott-ettinger** [comment]
Yes, the tensors are fixed to a size of 128, but some may be masked out with the invalid flag per time step.  The number of actual objects in the scene varies between scenarios.
